src/Init/Prelude.lean

```diff
@@ -806,6 +806,12 @@ decidability instance instead of the proposition, which has no code).
 
 If a proposition `p` is `Decidable`, then `(by decide : p)` will prove it by
 evaluating the decidability instance to `isTrue h` and returning `h`.
+
+Because `Decidable` carries data,
+when writing `@[simp]` lemmas which include a `Decidable` instance on the LHS,
```

Comment:
Is the corollary here that you can no longer use the lemma with `←` if no `Decidable` instance is on the *R*HS? And so in those cases, you need to write two versions of the same lemma?

(Re-posted as this got lost as a non-PR comment)